src/auth0/Auth.js

```diff
@@ -0,0 +1,91 @@
+import auth0 from 'auth0-js';
+import { message } from 'antd';
+import Cookies from 'js-cookie';
+
+import history from '../components/history/History';
+import { AUTH0 }from './constants';
+
+const { AUDIENCE, CLIENTID, DOMAIN, REDIRECTURI, RESPONSETYPE, SCOPE } = AUTH0;
+
+export default class Auth {
+
+    auth0 = new auth0.WebAuth({
+        audience: AUDIENCE,
+        clientID: CLIENTID,
+        domain: DOMAIN,
+        redirectUri: REDIRECTURI,
+        responseType: RESPONSETYPE,
+        scope: SCOPE,
+    });
+
+    constructor() {
+        this.login = this.login.bind(this);
+        this.logout = this.logout.bind(this);
+        this.handleAuthentication = this.handleAuthentication.bind(this);
+        this.isAuthenticated = this.isAuthenticated.bind(this);
+        this.getTokens = this.getTokens.bind(this);
+        this.renewSession = this.renewSession.bind(this);
```

Comment:
do these functions need to be bound?

Reply:
no, there is no need for binding. thanks for the catch.